apps/api/src/config/emailConfig.ts

```
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2025 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { z } from 'zod';
+import EnvironmentConfigError from './EnvironmentConfigError.ts';
+
+export const emailConfigSchema = z.object({
+	IS_PROD: z.string().optional().default('false'),
+	PORT: z.coerce.number().optional().default(3000),
+	UI_HOST: z.string().url().optional().default('http://localhost:5173'),
+	EMAIL_HOST: z.string().default('localhost'),
+	EMAIL_PORT: z.coerce.number().optional().default(1025),
+	EMAIL_FROM_ADDRESS: z.string().email().optional().default('noreply@pcgl.ca'),
```

not sure if doing defaults in the schema validation like this is a good idea, if we follow how ServerConfig works, we should just throw an error if they're not filled out otherwise the server could be throwing errors silently.